crates/rethnet_evm/src/db/remote.rs

```diff
@@ -0,0 +1,75 @@
+use tokio::runtime::{Builder, Runtime};
+
+use rethnet_eth::remote::RpcClient;
+use rethnet_eth::{Address, B256, U256};
+
+use revm::{db::DatabaseRef, AccountInfo, Bytecode};
+
+#[derive(std::hash::Hash, std::cmp::Eq, std::cmp::PartialEq)]
+struct StorageLocation {
```

Comment:
Is this used?

Reply:
nope! must have been part of some previous draft! thanks! removed in the latest version of my commit.